### ISSUE TITLE
Add Share app option inside app

### ISSUE THREAD

**Issue by ujjwalagrawal17** [commons-app/apps-android-commons#1284]
**Summary:** 

There should be an option to share the app.

**Steps to reproduce:** 

1. Add option to share in navigation drawer.
2. On Click of share option -
```
Intent sendIntent = new Intent();
sendIntent.setAction(Intent.ACTION_SEND);
sendIntent.putExtra(Intent.EXTRA_TEXT, "sharetext");
sendIntent.setType("text/plain");
startActivity(Intent.createChooser(sendIntent, getString(R.string.share)));
```			
**Expected behavior:** 
There should be an option to share the app.

**Observed behavior:** 
There is no option to share the app.

 **Commons app version:** 
2.6.7

**Comment by ujjwalagrawal17** [comment]
Share text can be 
**Upload photos to Wikimedia Commons using the app. 
Download Now: Google play link** 

I need some suggestions for the text

**Comment by maskaravivek** [comment]
This enhancement looks good to me. Let's wait for others to pitch in before starting work on it. :)

**Comment by misaochan** [comment]
Looks good to me, but I would probably put this in About instead of the nav drawer. Nav drawer is getting quite full. ;)

**Comment by nicolas-raoul** [comment]
Could you please post screenshots of a few popular apps that have a similar feature? Thanks!

**Comment by ujjwalagrawal17** [comment]
<table>
<tr>
<td>

![screenshot_20180312-225425](https://user-images.githubusercontent.com/19607555/37312801-83c747c2-2672-11e8-9055-6e5b3d77bf0c.png)

</td>
<td>

![screenshot_20180313-040737](https://user-images.githubusercontent.com/19607555/37313183-14cfe0ac-2674-11e8-915a-cee67dd47489.png)

</td>
</tr>
</table>

Screenshots from Phimp.me

**Comment by aniket1897** [comment]
I have worked on this issue can I submit PR?

**Comment by ujjwalagrawal17** [comment]
@aniket1897 I wanted to work on it. You should ask before working on the issue.
But since you had already worked on it. You submit the PR.

**Comment by aniket1897** [comment]
Sorry @ujjwalagrawal17 new to open source , next time I will take care.

**Comment by maskaravivek** [comment]
@nicolas-raoul @misaochan Your opinions on whether we need this feature? :)

**Comment by misaochan** [comment]
I definitely think it should NOT be in the navigation drawer the way the app in the screenshot does it. Feels a lot like feature bloat and users might get annoyed, since it isn't really something they would use often.

However, I think it could be nice to have a little "Share" icon in About. Perhaps next to the FB etc icons?

**Comment by maskaravivek** [comment]
Keeping the share button in the top bar (`Toolbar`) of the `About` page also seems to be a good idea to me. That way it won't get lost in the already overcrowded `About` page.

**Comment by misaochan** [comment]
@maskaravivek Sounds like a plan! 👍

**Comment by wailbabou** [comment]
@maskaravivek  I start working on this

**Comment by ujjwalagrawal17** [comment]
@wailbabou we already have a PR on this issue by @aniket1897

**Comment by wailbabou** [comment]
@ujjwalagrawal17  but there is no comments from him since 5 days ! , and you said :[ If it takes more than two days, please post other comments at least every two days]

**Comment by Mansi449** [comment]
Hi @ujjwalagrawal17, I would like to work on this issue

**Comment by nicolas-raoul** [comment]
@aniket1897 if you have made any progress on this please tell us, thanks!

**Comment by aniket1897** [comment]
Yes I am working on the issue

**Comment by Mansi449** [comment]
@aaronpp65 
I have already opened a pull request for this :)

**Comment by Mansi449** [comment]
Please do review my PR
